master/security/namespace-policy.md

```diff
@@ -1,15 +1,15 @@
 ---
-title: Use namespace in policy rules
+title: Use namespace in policy
 description: Use namespaces and namespaceSelectors in Calico network policy to group or separate resources. Use policy rules to allow or deny traffic to/from pods belonging to specific namespaces.
 ---
 
 ### Big picture
 
-Use {{site.prodname}} network policy rules to reference pods in other namespaces.
+Use {{site.prodname}} network policies to reference pods in other namespaces.
 
 ### Value
 
-Kubernetes namespaces let you group/separate resources to meet a variety of use-cases. For example you can use namespaces to separate development, production, and QA environments, or to allow different teams to use the same cluster. Using namespace selectors in {{site.prodname}} policy rules allows you to allow or deny traffic to/from pods belonging to specific namespaces. 
+Kubernetes namespaces let you group/separate resources to meet a variety of use-cases. For example you can use namespaces to separate development, production, and QA environments, or to allow different teams to use the same cluster. Using namespace selectors in {{site.prodname}} network policies allows you to allow or deny traffic to/from pods belonging to specific namespaces.
```

Comment:
Kubernetes namespaces let you group/separate resources to meet a variety of use cases. For example, you can use namespaces to separate development, production, and QA environments, or to allow different teams to use the same cluster. You can use namespace selectors in {{site.prodname}} network policies to allow or deny traffic to/from pods belonging to specific namespaces.

Reply:
Fixed